packages/devtools/lib/src/memory/memory.dart

```diff
@@ -204,7 +282,10 @@ class MemoryScreen extends Screen {
   CoreElement createLiveChartArea() {
     final CoreElement container = div(c: 'section perf-chart table-border')
       ..layoutVertical();
-    memoryChart = MemoryChart(container);
+    memoryChart = MemoryChart(this);
+
+    container.add(memoryChart);
+
     memoryChart.disabled = true;
```

Comment:
style nit: move the container.add call to after the memory chart is set to be disabled.

Reply:
memoryChart = MemoryChart(this)..disabled = true;
container.add(memoryChart);

done.